adr/0021-rules-for-entity-mapping-in-service-catalog.md

```diff
@@ -0,0 +1,175 @@
+# Rules for entity mapping in Service Catalog
+
+* Status: Proposed
+* Deciders: mimotej, SamoKopecky, tumido
+* Date: 2022-07-26
+
+## Context and Problem Statement
+
+Backstage offers multiple resource types which can be used to represent entities and their relations in service catalog. We need to have a set of rules present in Operate First which will provide context and guidance on how entities are mapped to Backstage resources.
```

Comment:
I think we should call the "resource types" "entity types" since they are called like that in the [backstage documentation](https://backstage.io/docs/features/software-catalog/descriptor-format#overall-shape-of-an-entity). I can also see the name "resource" being a bit confusing because right now there is a resource that has the `Kind: Resource`.